app/collections/tweets/report.rb

```diff
@@ -0,0 +1,47 @@
+class Tweets::Report
+  MONTH_FORMAT = '%m.%Y'.freeze
+
+  def initialize(organization, month)
+    @organization = organization
+    @month = month
+  end
+
+  def generate_report
+    report = ''
+    tweets_by_week.each do |week_title, days|
+      report << "Week: #{week_title}\n"
+      days.each do |day_title, tweets|
+        report << " Day: #{day_title}\n"
+        if !tweets.empty?
```

Comment:
for such cases, when you have an else block, prefer inverting the condition or asking if it is present: `tweets.present?`, so that there is no negation in the condition